server/src/data/channels-data.js

```diff
@@ -71,6 +71,68 @@ class ChannelsData {
     return _.get(channel, 'creator');
   }
 
+  /**
+   * @public
+   * @param {object} options
+   * @param {number} [options.offset]
+   * @param {number} [options.limit]
+   * @param {object} [options.restrictions]
+   * @return {Promise<*|Array<Model>>}
+   */
+  async getAll(options = {}) {
```

Comment:
given that this is already in data, can we replace the ``options`` input with a specific hash of keys, like, 
``getAll({offset=0, limit=10})``. Not sure exactly what getRestrictionsSync needs though.

Reply:
Yes, you can write code this way but believe it is not safe and not flexible. Just image that function gets "string" as param.